.github/workflows/test.yml

```
@@ -23,13 +23,17 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@ff7abcd0c3c05ccf6adc123a8cd1fd4fb30fb493
-      - name: Run docker compose
+      - name: Launch Splunk Docker instance
         run: SPLUNK_VERSION=${{ matrix.splunk-version }} docker compose up -d
-      - name: Setup Python
+      - name: Setup Python ${{ matrix.python-version }}
         uses: actions/setup-python@e797f83bcb11b83ae66e0230d6156d7c80228e7c
         with:
           python-version: ${{ matrix.python-version }}
-      - name: Install tox
-        run: pip install tox
-      - name: Test Execution
-        run: tox -e py -- ./tests
+      - name: (Python 3.7) Install dependencies
+        if: ${{ matrix.python-version == '3.7' }}
+        run: python -m pip install python-dotenv deprecation pytest
```

```suggestion
        run: python -m pip install python-dotenv pytest
```

Can we drop that now? Or is it still required by something?

---

Done!